resources/app/log-collector.yaml

```diff
@@ -0,0 +1,148 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: log-collector
+  namespace: kube-system
+  labels:
+    app: log-collector
+spec:
+  type: ClusterIP
+  ports:
+    - name: rest
+      protocol: TCP
+      port: 8083
+  selector:
+    app: log-collector
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: log-forwarders
+  namespace: kube-system
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: log-collector
+  namespace: kube-system
+  labels:
+    app: log-collector
+data:
+  adapter.json: |
+    {
+      "Gravity": {
+        "ApiListenAddr": "0.0.0.0:8083",
+        "Kubernetes": {
+          "Namespace": "kube-system",
+          "ForwarderConfigMapName": "log-forwarders"
+        }
+      },
+
+      "Logrange": {
+        "Partition": "logrange.pipe=__default__",
+        "ForwarderTmplFile": "/opt/logrange/gravity/config/forward-tmpl.json",
+        "CronQueries": [
+          {
+            "Query": "truncate minsize 4mb maxsize 500mb maxdbsize 10gb",
+            "PeriodSec": 60
+          }
+        ],
+        "Kubernetes": {
+          "Namespace": "kube-system",
+          "ForwarderConfigMap": "lr-forwarder"
+        },
+        "Transport": {
+          "ListenAddr": "lr-aggregator.kube-system.svc.cluster.local:9966",
+          "TlsEnabled": false,
+          "Tls2Way": false,
+          "TlsCertFile": "/opt/logrange/gravity/config/tls/client.crt",
```

Comment:
Where do these certificate/key/ca come from? Same q for all other deployments.

Reply:
for now TlsEnabled=false and those Tls related files are not read, it's just an example how they can be specified here. How exactly they will appear in container (the safe way), this is a good question to figure out, easiest (and not safe) just mount them...

Reply:
We need to turn TLS on. All components inside the cluster communicate using mutual TLS.

Reply:
@r0mant please let me know how you do it for your existing services, how certs appear in the containers? For now it looks like we need to mount them during container start... (so we need to know from where to take them)... Might be we better have a call to discuss this...